library/src/main/java/eu/inloop/viewmodel/IView.java

```diff
@@ -1,5 +1,7 @@
 package eu.inloop.viewmodel;
 
-public interface IView {
+import eu.inloop.viewmodel.binding.ViewModelBindingConfig;
 
+public interface IView {
+    ViewModelBindingConfig getViewModelBindingConfig();
```

Comment:
Add javadoc here and @Nullable 
I would also explain to the developer that he can return null in case binding is not used.